pages/restake/bridge.mdx

```diff
@@ -0,0 +1,78 @@
+import Callout from "/components/Callout";
+
+# Bridge
+
+In order to participate in Tangle's restaking infrastructure, users need to first bridge in their assets from EVM-based networks such as Ethereum into Tangle. For this, we have a dedicated bridge & its corresponding DApp page.
+
+[Access Tangle DApp's Bridge page here](https://app.tangle.tools/bridge)
+
+## Supported Assets
+
+As of the time of writing of this page, the bridge only supports WETH for testnet transfers between Holesky and Tangle.
+
+Soon, users will be able to create and deploy a Hyperlane Warp Route blueprint instance to support additional assets. [Learn more about Blueprints here](/developers/blueprints).
+
+More information about adding support for additional assets will be provided in the near future. For now, make sure you've joined our Discord server & Telegram channel to stay updated.
+
+TODO: For custom bridging, how will the users choose what bridge to use if anyone can deploy their warp route instances? The list will need to be of addresses, but those addresses will need to be registered in a specific repo. We don't have that repo yet, so for now we depend on the Hyperlane registry repo in which users can submit their PRs to have their own instances included.
+
+## How the Bridge Works
+
+Bridging from EVM-based blockchains into Tangle EVM works by leveraging Hyperlane, a permissionless interoperability layer that allows smart contract developers to send arbitrary data between blockchains. [Learn more about Hyperlane here](https://docs.hyperlane.xyz/docs/protocol/protocol-overview).
+
+## Bridge Fees
+
+Apart from the network transaction fees (gas fees), ...
+
+Hyperlane warp route fees?
+Are there multiple transactions that need to be made by the user? 2? Do each charge a fee?
+
+## How to Use the Bridge
+
+### Step 1: Access Tangle DApp & Connect Wallet
+
+- Open [Tangle DApp's Bridge page](https://app.tangle.tools/bridge).
+- Connect your wallet to the DApp by clicking on the **Connect Wallet** button on the top right and selecting your preferred wallet provider.
+
+TODO: Should the user/wallet selected always be EVM-based?
+
+### Step 2: Select the Source & Destination Networks
+
+TODO: Explain that when bringing IN asssets, source chain should never be Tangle, and then when bringing OUT assets, destination chain should never be Tangle.
```

Comment:
This is aggressive, I prefer positive language over negative. Just say it simply.
- When bridging in to Tangle, select Tangle as the destination chain.
- When bridging out of Tangle, select Tangle as the source chain.

Reply:
@drewstone The TODOs are meant to be resolved within this PR, just a way to note down my thoughts on what's left for the sections